extensions/solr/deployment/src/main/java/org/apache/camel/quarkus/component/solr/deployment/SolrProcessor.java

```diff
@@ -65,6 +66,8 @@ void registerForReflection(BuildProducer<ReflectiveClassBuildItem> reflectiveCla
                     }
                 })
                 .forEach(reflectiveClass::produce);
+
+        reflectiveClass.produce(new ReflectiveClassBuildItem(false, false, ClientCnxnSocketNIO.class.getName()));
```

Comment:
We are registering the same class in HBaseProcessor. This is perhaps not enough for a separate support extension yet.